hw1/02-palindrome.js

```diff
@@ -1,3 +1,33 @@
-const elem = document.querySelector('input');
+function isPalindrome(value) {
+  const stringInput = value.toString();
+  const reversedInput = stringInput.split("").reverse().join("");
```

Comment:
Rule [19.6](https://github.com/airbnb/javascript#whitespace--chains) states that long method chains(2 or more) should be broken up with indentation, with a leading dot.